### PR TITLE
fix ordering hex20 and fixed support wedge15

### PR DESCRIPTION
**Pull request by ffilotto**

### CODE REVIEW

**Comment by nschloe** [comment]
You need to run `make format` to get the formatting right.

File: meshio/gmsh/common.py [hunk 1]
```diff
@@ -152,9 +153,12 @@ def _gmsh_to_meshio_order(cells):
         # fmt: off
         "tetra10": [0, 1, 2, 3, 4, 5, 6, 7, 9, 8],
         "hexahedron20": [
-            0, 1, 2, 3, 4, 5, 6, 7, 8, 11, 16,
-            9, 17, 10, 18, 19, 12, 15, 13, 14
-        ],
+            0, 1, 2, 3, 4, 5, 6, 7, 8, 11, 13,
+            9, 16, 18, 19, 17, 10, 12, 14, 15,
+        ],  # https://vtk.org/doc/release/4.2/html/classvtkQuadraticHexahedron.html
```
Comment (nschloe) [inline meshio/gmsh/common.py:158]: These are links to VTK, but you're manipulating the Gmsh format. Perhaps you're editing the wrong file?

**Comment by nschloe** [comment]
@ffilotto Let me know if you need help.

File: meshio/gmsh/common.py (hunk 1 above)
Comment (ffilotto) [reply meshio/gmsh/common.py:158]: as I understood, meshio follows VTK conventions for cell numbering, or am I wrong? I got this numbering from comparison with the corresponding cell in here: http://www.manpagez.com/info/gmsh/gmsh-2.2.6/gmsh_65.php#SEC65

So what I am doing is to give the re ordering from gmsh (gmsh format, see link above) and meshio/vtk (see link in PR)

**Comment by ffilotto** [comment]
> 
> 
> You need to run `make format` to get the formatting right.

I am on windows and still need to figure out out to "make" it. Do you know if "make" comes with Git for Windows?

**Comment by keurfonluu** [comment]
> > You need to run `make format` to get the formatting right.
> 
> I am on windows and still need to figure out out to "make" it. Do you know if "make" comes with Git for Windows?

You can simply run:

```
isort -rc .
black .
```

which are executed by `make format`. You have to install `black` and `isort` beforehand:

```
pip install black
pip install isort
````

File: meshio/gmsh/common.py (hunk 1 above)
Comment (nschloe) [reply meshio/gmsh/common.py:158]: That's a nice link! Can you add it to the code too? (Otherwise I'll forget about it.)

**Pull request merged**